README.md

```diff
@@ -58,3 +58,51 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
 
 _**Note**: The new tabs opened using the gamepad navigator will use [Google](https://www.google.com/) as the homepage and
 override the default new tab page. However, it can be configured to open any other website instead of Google._
+
+## Default Controls
+
+<p align="center">
+   <img src="src/images/gamepad.svg">
+   <i>
+      <p align="center">
+         Image Source: <a href="https://tinyurl.com/y2wvtldg">W3C Gamepad API Documentation</a>
+      </p>
+   </i>
+</p>
+
+Although the gamepad inputs are reconfigurable, the extension provides a default configuration for each gamepad input.
+The default action for each gamepad input is given below.<br>
+(_Please refer to the above diagram for gamepad inputs_)
+
+| Gamepad Input | Default Action | Speed Factor | Invert Action / Open a new tab or window in background |
+| :---: | :--- | :---: | :---: |
+| `Button 0` | Click | - | - |
+| `Button 1` | None | - | - |
+| `Button 2` | History back button | - | - |
```

Comment:
It would be helpful to link each action to its section in the documentation.

Reply:
Hi @the-t-in-rtf, thanks for the feedback. I'm not sure whether I understood this correctly, but do you mean that we should link the actions to the navigation invokers?

Reply:
You spelled out elsewhere what each action does.  Because you're now in a slightly different context, it'd be good to at least mention it once, something like "For more information on each action, read the related docs", linked to the appropriate page.

Reply:
@the-t-in-rtf as far as I could recall, the only place we described what each action does is in the corresponding invoker definitions. So maybe we should add the list of actions here, with a brief description. We can also add a reference to the invoker's definition along if you allow.

Reply:
That sounds like a good idea.